tests/test_ntp_server.rb

```diff
@@ -83,4 +92,54 @@ def test_multiple
     refute_includes(Cisco::NtpServer.ntpservers, id1)
     refute_includes(Cisco::NtpServer.ntpservers, id2)
   end
+
+  def test_domain_name
+    id = '0.us.pool.ntp.org'
```

Comment:
When I run this on my testbed it fails because we are behind a proxy.  This is fine for now, but I will probably move this into a yaml configuration file that automatically skips the test if the config file is unavailable and then anyone that wants to run this test can add any internal or external servers